.gitlab/deploy_packages/deploy_common.yml

```diff
@@ -79,12 +79,22 @@ deploy_packages_oci:
   needs: [ "packaging_oci" ]
   tags: ["arch:amd64"]
   stage: deploy_packages
-  rules:
-    !reference [.on_deploy_a7]
   before_script:
     - ls $OMNIBUS_PACKAGE_DIR
+  rules:
+    !reference [.on_deploy_a7]
   script:
-    - $S3_CP_CMD --recursive --exclude "*" --include "datadog-agent-*oci.tar" "$OMNIBUS_PACKAGE_DIR" "$S3_RELEASE_ARTIFACTS_URI/oci/"
+    - python3 -m pip install -r tasks/libs/requirements-github.txt
+    - set +x
+    - export GITHUB_KEY_B64=$($CI_PROJECT_DIR/tools/ci/aws_ssm_get_wrapper.sh ci.datadog-agent.platform-github-app-key)
+    - export GITHUB_APP_ID=682216
+    - export GH_TOKEN=$(inv -e github.get-token-from-app --app-id-env=GITHUB_APP_ID --pkey-env=GITHUB_KEY_B64)
+    - export VERSION=$(inv agent.version -u)
+    - git config --global credential.helper '!f() { echo username=x-access-token; echo "password=$GH_TOKEN"; };f'
+    - git clone https://github.com/DataDog/datadog-packages /tmp/datadog-packages
+    - cd /tmp/datadog-packages/cmd/datadog-package
+    - go build .
+    - ./datadog-package push registry.ddbuild.io/ci/datadog-agent/datadog-agent-ociru:${VERSION} ${OMNIBUS_PACKAGE_DIR}/datadog-agent-${MAJOR_VERSION}.*.oci.tar
```

Comment:
<!-- Suggestions are specific requests to improve the subject under review. It is assumed that we all want to do what's best, so these comments are never dismissed as “mere suggestions”, but are taken seriously.  -->
**💬 Suggestion:** ‏Can we use `registry.ddbuild.io/ci/remote-updates/datadog-agent:${VERSION}`? That'd match the tracers

Reply:
For `registry.ddbuild.io/ci/remote-updates/` no problem at all on my end.

Regarding the image name itself, I believe @vboulineau wanted to have a more descriptive name than `datadog-agent`.
The prefix/folder might be enough to disambiguate what kind of `datadog-package` though

Reply:
True, the name may change again, but it's easy to change.